src/index.js

```diff
@@ -46,22 +47,19 @@ function rotateXYByAngle(pos, angle) {
   return [x, y]
 }
 
-const getTouchHandlerOption = props => {
-  if (props.touchHandlerOption) return props.touchHandlerOption
-  return props.preventDefaultTouchmoveEvent
-    ? { passive: false }
```

Comment:
`eventListener` options no longer needed since this pr attaches touch listeners to the swipe container via `ref` callback